src/ray/protobuf/node_manager.proto

```diff
@@ -219,6 +219,8 @@ message ObjectStoreStats {
   // the node has more pull requests than available object store
   // memory.
   bool object_pulls_queued = 13;
+  // The number of primary copies of objects in the local node.
+  int64 num_object_store_primary_copies = 14;
```

Comment:
So this number is for objects owned by this raylet and the above number is for objects owned by individual worker?

Reply:
@lanbochen-anyscale  

> this number is for objects owned by this raylet

primary copy of the objects in this raylet

> number is for objects owned by individual worker

Yes, you need to aggregrate it in cli side.